draft-ietf-mmusic-t140-usage-data-channel.xml

```diff
@@ -70,7 +70,7 @@
         see Section 3.2 of <xref target="I-D.ietf-rtcweb-data-channel"/>.
       </t>
       <t>
-        The brief notation "T.140" is used as a synonym for the text
+        The brief notation "T.140" is used as an acronym for the text
```

Comment:
Acronym should be a name composed by the initial letters of a sequence of words. How about just "a name" instead?

Reply:
I can use "a name". I am sure the RFC editor will let us know if he/she thinks there is something better.